caddy/mercure-skip.go

```diff
@@ -2,9 +2,12 @@
 
 package caddy
 
-import (
-	"github.com/caddyserver/caddy/v2"
```

Comment:
I believe this is needed to compile with `nomecure` tags? At least, I got a missing import error when trying to compile without it on my machine.